sentry-logback/src/main/java/io/sentry/logback/SentryAppender.java

```diff
@@ -162,6 +170,26 @@ protected void append(@NotNull ILoggingEvent eventObject) {
     return event;
   }
 
+  /**
+   * Captures a Sentry log from Logback's {@link ILoggingEvent}.
+   *
+   * @param loggingEvent the logback event
+   */
+  // for the Android compatibility we must use old Java Date class
+  @SuppressWarnings("JdkObsolete")
+  protected void captureLog(@NotNull ILoggingEvent loggingEvent) {
+    final @NotNull SentryLogLevel sentryLevel = toSentryLogLevel(loggingEvent.getLevel());
```

Comment:
let's also check if logs is enabled here, so we don't format the string if not necessary?